app/Platform/Commands/AddStandaloneSystem.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * @deprecated - remove after this executes once on prod
+ */
+class AddStandaloneSystem extends Command
+{
+    protected $signature = 'ra:platform:system:add-standalones';
+    protected $description = 'Set up the Standalones system in the database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle(): void
+    {
+        $this->info('Adding the Standalones system to the DB.');
+
+        $standalonesConfig = $this->loadStandalonesConfig();
+
+        if (!$standalonesConfig) {
+            $this->error('Something went wrong loading the Standalones system config.');
+
+            return;
+        }
+
+        $this->insertStandalonesSystemToDb($standalonesConfig);
+
+        $this->info('Added the Standalones system (' . $standalonesConfig['id'] . ') to the DB.');
+    }
+
+    private function loadStandalonesConfig(): ?array
+    {
+        $standalonesConfig = null;
+
+        $systemsConfig = config("systems");
+        foreach ($systemsConfig as $id => $config) {
+            if (isset($config['name']) && $config['name'] === 'Standalones') {
+                $standalonesConfig = $config;
+                $standalonesConfig['id'] = $id;
+
+                break;
+            }
+        }
+
+        return $standalonesConfig;
+    }
+
+    private function insertStandalonesSystemToDb(array $standalonesConfig): void
+    {
+        $doesStandalonesSystemExist = DB::table('Console')->where('ID', $standalonesConfig['id'])->exists();
+        if (!$doesStandalonesSystemExist) {
+            DB::table('Console')->insert([
+                'ID' => $standalonesConfig['id'],
+                'Name' => $standalonesConfig['name'],
+            ]);
+        }
+    }
+}
```

Comment:
Wouldn't it be better to do this with a migration?

Is there going to be another ID other than 102?

Reply:
To the best of my knowledge, none of our migrations write rows to the DB. I vaguely recollect a conversation that happened on another PR where we decided not to take that approach, though I can't remember the PR anymore.

As I understand our implementation plan, the new system will only have a single system ID.

Reply:
You're thinking about the Messages PR. That was a hefty chunk of logic that had to be executed. This is just inserting a single row into the DB. For previous new systems, that's always just been done by hand some time before deploying the code.

Reply:
Migration added in latest and command removed.